WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/RefundShippingLabelViewModel.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import struct Yosemite.ShippingLabel
+
+/// View model for `RefundShippingLabelViewController` that provides data that are ready for display in the view.
+struct RefundShippingLabelViewModel {
+    let purchaseDate: String
+    let refundableAmount: String
+    let refundButtonTitle: String
+
+    init(shippingLabel: ShippingLabel, currencyFormatter: CurrencyFormatter) {
+        self.purchaseDate = shippingLabel.dateCreated.toString(dateStyle: .medium, timeStyle: .short)
+        self.refundableAmount = currencyFormatter.formatAmount(Decimal(shippingLabel.refundableAmount), with: shippingLabel.currency) ?? ""
+        self.refundButtonTitle = String.localizedStringWithFormat(Localization.refundButtonTitleFormat, refundableAmount)
```

Comment:
Do you think it's worth it to unit test these string formats?

Reply:
yup great point! added in https://github.com/woocommerce/woocommerce-ios/pull/3306/commits/a4d9c1f7c05a73b0a134a39dec5f545b8af1c38d